src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -355,16 +357,17 @@ private void setup(OptionSet options, Properties properties) {
 		this.pubsubTopic 		 = fetchOption("pubsub_topic", options, properties, "maxwell");
 		this.ddlPubsubTopic  = fetchOption("ddl_pubsub_topic", options, properties, this.pubsubTopic);
 
-		this.rabbitmqHost           = fetchOption("rabbitmq_host", options, properties, "localhost");
-		this.rabbitmqUser			= fetchOption("rabbitmq_user", options, properties, "guest");
+		this.rabbitmqHost           		= fetchOption("rabbitmq_host", options, properties, "localhost");
+		this.rabbitmqPort 			= Integer.parseInt(fetchOption("rabbitmq_port", options, properties, "5672"));
```

Comment:
What will happen if I pass rabbitmq_port as say "abcd" i.e. invalid port or say "123456789" i.e. port, not in valid range?